measure.go

```diff
@@ -143,6 +143,10 @@ func (m *measure) Put(key datastore.Key, value []byte) error {
 	return err
 }
 
+func (m *measure) Sync(prefix datastore.Key) error {
+	return nil
```

Comment:
This needs to call sync on the underlying datasture.

It should also add a new metric for sync calls.

Reply:
Yep, was going to fix this but got distracted with #24

Reply:
better?